searchcore/src/tests/proton/matching/matching_test.cpp

```diff
@@ -327,10 +327,14 @@ struct MyWorld {
         }
     };
 
+    struct DummyAttributeExecutor : public IAttributeExecutor {
+        void asyncForAttribute(const vespalib::string &, std::shared_ptr<IAttributeFunctor>) const override {}
+    };
     void verify_diversity_filter(SearchRequest::SP req, bool expectDiverse) {
         Matcher::SP matcher = createMatcher();
         search::fef::Properties overrides;
-        auto mtf = matcher->create_match_tools_factory(*req, searchContext, attributeContext, metaStore, overrides);
+        DummyAttributeExecutor attrExec;
+        auto mtf = matcher->create_match_tools_factory(*req, searchContext, attributeContext, attrExec, metaStore, overrides);
```

Comment:
maybe the attribute executor should be part of the attribute context?

Reply:
I will try and see if that is doable. I hoped that might be an option.